assets/src/edit-story/app/media/media3p/useFetchCategoriesEffect.js

```diff
@@ -29,6 +29,17 @@ import { useSnackbar } from '../../snackbar';
 import { useMedia3pApi } from './api';
 import { PROVIDERS } from './providerConfiguration';
 
+/**
+ *
```

Comment:
May as well add a once sentence introduction.
Something like 'The side effect that fetches the media categories from the backend.'.

Reply:
Done